scipy/spatial/distance.py

```diff
@@ -1143,18 +1217,17 @@ def pdist(X, metric='euclidean', p=None, w=None, V=None, VI=None):
         'jaccard', 'kulsinski', 'mahalanobis', 'matching',
         'minkowski', 'rogerstanimoto', 'russellrao', 'seuclidean',
         'sokalmichener', 'sokalsneath', 'sqeuclidean', 'yule'.
-    p : double, optional
-        The p-norm to apply
-        Only for Minkowski, weighted and unweighted. Default: 2.
-    w : ndarray, optional
-        The weight vector.
-        Only for weighted Minkowski. Mandatory
-    V : ndarray, optional
-        The variance vector
-        Only for standardized Euclidean. Default: var(X, axis=0, ddof=1)
-    VI : ndarray, optional
-        The inverse of the covariance matrix
-        Only for Mahalanobis. Default: inv(cov(X.T)).T
+    args, kwargs : tuple and dict, optional
+        Extra arguments to `metric`: refer to each metric documentation for a
+        list of all possible arguments.
+        NOTE: Always pass the arguments as `kwargs` since `args` is deprecated
+        and will be removed in a future version.
```

Comment:
I am not sure to understand what are you suggesting...something like this?

    p : double, Deprecated: pass as kwarg instead
        The p-norm to apply
        Only for Minkowski, weighted and unweighted. Default: 2.
    w : ndarray, Deprecated: pass as kwarg instead
        The weight vector.
        Only for weighted Minkowski. Mandatory
    V : ndarray, Deprecated: pass as kwarg instead
        The variance vector
        Only for standardized Euclidean. Default: var(X, axis=0, ddof=1)
    VI : ndarray, Deprecated: pass as kwarg instead
        The inverse of the covariance matrix
        Only for Mahalanobis. Default: inv(cov(X.T)).T
    kwargs : dict, optional
        Extra arguments to `metric`: refer to each metric documentation for a
        list of all possible arguments.